pgtoolkit/ctl.py

```diff
@@ -246,3 +254,57 @@ def status(self, datadir: Union[Path, str]) -> Status:
         """
         cmd = [str(self.pg_ctl), "status"] + ["-D", str(datadir)]
         return Status(self.run_command(cmd).returncode)
+
+
+def num_version(text_version: str) -> int:
+    """Return PostgreSQL numeric version as defined by LibPQ PQserverVersion
+    >>> num_version('pg_ctl (PostgreSQL) 9.6.3')
+    90603
+    >>> num_version('pg_ctl (PostgreSQL) 9.2.0')
+    90200
+    >>> num_version('pg_ctl (PostgreSQL) 11.10')
+    110010
+    >>> num_version('pg_ctl (PostgreSQL) 11.1')
+    110001
+    """
+    res = re.match(
+        r"pg_ctl \((PostgreSQL|EnterpriseDB)\) ([0-9]+)\.([0-9]+)(?:\.([0-9]+))?",
```

Comment:
I wonder if we could just match `\(\w+\)` because we do not actually use the "vendor" name.